src/FirebirdSql.Data.FirebirdClient/Common/NativeHelpers.cs

```diff
@@ -16,18 +16,33 @@
 //$Authors = Jiri Cincura (jiri@cincura.net)
 
 using System;
+using System.Collections.Concurrent;
 
 namespace FirebirdSql.Data.Common;
 
 internal static class NativeHelpers
 {
-	public static void CallIfExists(Action action)
+	private static readonly ConcurrentDictionary<string, bool> _cache = new ConcurrentDictionary<string, bool>();
```

Comment:
Use explicit comparer (ordinal) for even better perf.

Reply:
Fixed